f/connectors/alerts/alerts_gcs.py

```diff
@@ -516,69 +567,57 @@ def handle_output(self, geojsons, metadatas):
 
                         except errors.UniqueViolation:
                             logger.info(
-                                f"Skipping insert due to UniqueViolation, this alert has been processed already in the past: {source}"
+                                f"Skipping insert due to UniqueViolation, this entry has been processed already in the past: {_id}"
```

Comment:
If I follow, this never would have triggered before your change. There was not a unique constraint before this.

Reply:
Yes, that is correct! This was an oversight already in the frizzle-dagster implementation, resulting in https://github.com/ConservationMetrics/frizzle-dagster/issues/139